src/app/icd/ICDManager.cpp

```diff
@@ -275,15 +275,6 @@ void ICDManager::UpdateOperationState(OperationalState state)
 
         System::Clock::Milliseconds32 slowPollInterval = ICDConfigurationData::GetInstance().GetSlowPollingInterval();
 
-#if ICD_ENFORCE_SIT_SLOW_POLL_LIMIT
```

Comment:
Do we not need this check anymore for SIT?

Reply:
Logic was moved to the ICDConfigurationData class but some trailing code was left in the ICDManager.
This removes what was not removed when ICDConfigurationData was added.